log-generator/log-generator.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+process.env.NEW_RELIC_NO_CONFIG_FILE = true
+process.env.NEW_RELIC_APP_NAME = 'log-generator'
+process.env.NEW_RELIC_LICENSE_KEY = 'fill-in-license-key'
```

Comment:
we should included `NEW_RELIC_HOST` and default it to `staging-collector.newrelic.com` to avoid logging this data to prod